op-program/Makefile

```diff
@@ -24,23 +24,8 @@ op-program-client-mips:
 	# verify output with: readelf -h bin/op-program-client.elf
 	# result is mips32, big endian, R3000
 
-op-program-client-wasm:
-	env GO111MODULE=on GOOS=js GOARCH=wasm go build -gcflags=all=-d=softfloat -v $(LDFLAGS) -o ./bin/op-program-client.wasm ./client/cmd/main.go
-	# env GO111MODULE=on GOOS=js GOARCH=wasm go build  -v $(LDFLAGS) -o ./bin/op-program-client.wasm ./client/cmd/main.go
-	# tinygo build -o ./bin/op-program-client.wasm -target=wasm ./client/cmd/main.go
-
-op-program-client-wasm-preimage:
-	env GO111MODULE=on GOOS=js GOARCH=wasm go build -gcflags=all=-d=softfloat -v $(LDFLAGS) -o ./bin/op-program-client-preimage.wasm ./client-zk/cmd/main.go
-
 op-program-client-wasi:
 	env GO111MODULE=on GOOS=wasip1 GOARCH=wasm go build -gcflags=all=-d=softfloat -v $(LDFLAGS) -o ./bin/op-program-client.wasi ./client/cmd/main.go
-	# tinygo build -o ./bin/op-program-client.wasm -target=wasm ./client/cmd/main.go
-
-op-program-client-wasi-preimage:
-	env GO111MODULE=on GOOS=wasip1 GOARCH=wasm go build -gcflags=all=-d=softfloat -v $(LDFLAGS) -o ./bin/op-program-client-preimage.wasi ./client-zk/cmd/main.go
-
-hostio-test:
-	env GO111MODULE=on GOOS=wasip1 GOARCH=wasm go build -gcflags=all=-d=softfloat -v $(LDFLAGS) -o ./bin/hostio-test.wasi ./hostio_test/main.go
```

Comment:
Minor:
The .wasi suffix is no different from .wasm. Essentially it is in binary format.
.wasi implies it seems a special case which is not true: you can still use wasmtime/node to handle it with no extra effort (despite providing a wasm env).

Reply:
But this is a style issue, not a big deal.

Reply:
Good points. I've changed all wasi related files to wasm to avoid confusing users. Fixed in https://github.com/ethstorage/optimism/pull/9/commits/2a0a9fae2fde90bb7107456afd0b62a8bbbf8db0